scripts/backup

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+#Keep this path for calling _common.sh inside the execution's context of backup and restore scripts
+source ../settings/scripts/_common.sh
+source /usr/share/yunohost/helpers
+
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
+ynh_clean_setup () {
+	ynh_clean_check_starting
+}
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
+
+#=================================================
+# LOAD SETTINGS
+#=================================================
+ynh_script_progression --message="Loading installation settings..." --weight=1
+
+app=$YNH_APP_INSTANCE_NAME
+
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
+domain=$(ynh_app_setting_get --app=$app --key=domain)
+db_name=$(ynh_app_setting_get --app=$app --key=db_name)
+#db_user=$(ynh_app_setting_get --app=$app --key=db_user)
+
+#=================================================
+# STANDARD BACKUP STEPS
+#=================================================
+# STOP SYSTEMD SERVICE
+#=================================================
+ynh_script_progression --message="Stopping a systemd service..." --weight=2
+
+ynh_systemd_action --service_name="$app" --action="stop" --log_path="/var/log/$app/$app.log"
```

Comment:
I know this is from the example_ynh, but as explained in [here](https://github.com/YunoHost/example_ynh/pull/116) I don't understand the benefit of stopping the service during backup (on the contrary, it's probably not relevant and will create service interruption for no reason)

Reply:
@alexAubin I am wondering if the backup script is only triggered by the upgrade script : I am questioning my use of `--line_match` on the backup script... This may not make sense here and make the backup slower

Also, we may want this warning on `package_check` to warn the opposite
```
Compare the syntax to the packaging template... [2/...]

Looks like there isn't many comment progression indicators in your script
Consider adding more indicators to help your users know the progression of the scripts
Here the culprit:
scripts/backup
```
(I wonder if anyone has passed this test with a success :)

Reply:
Uuuuh I'm not sure to understand your question ? My suggestion is that you should not even start/stop the service because (unless proven otherwise) there are only drawbacks in doing so and no benefit. The backup script is *- apart from DB dump which really 'do' something -* just a *declaration* of what's to be backuped and the *actual file copy* happens much later in the core itself... (so not during the backup script execution)

> Also, we may want this warning on package_check to warn the opposite

Well my opinion is that all these "template" checks were implemented very quickly without any proper discussion on their relevance and accuracy. On this specific topic I already tried to explain a couple years ago how having progress bars during backup script is pretty meaningless and misleading but with no success ... So to me that check simply makes no sense.